### DISCUSSION TITLE
Getting access to old organisation account

### DISCUSSION THREAD

**Discussion by AndreasEFlightright** [community/community#126692] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

hi there, 
some former employee of our company (cannot reach out to him anymore) created a github free organisation 2016 with the name of our company. 
As we were not actively using the account for many years (neither was he, as he opened it up most likely with some companies email related to our domain) now the entry login credentials are lost due to multiple changes of responsible persons and lack of good secret handling at the startup phase of the company. 
Now, as the new responsible manager I want to get access to the account as we are propability want to move our company repo from gitlab to github.
I opened up a normal account not accessable support ticket on my personal github account as no payed orga is available right now and the transfer ownership of organisation ticket cannot be created due to a bug within the related form of github. 
However, nobody at github is reacting to this ticket. In comparison to gitlab or AWS this seems like a real lag in support to the community to me. 
What to do next? Is there some normal 1st level support I could trigger to get stuff moving? 
What is your experience of support quality in general of github in comparison to AWS and/or gitlab?

**Comment by ebndev** [comment]
Hi @AndreasEFlightright,

We are no longer accepting requests to release dormant usernames. We are not able to accommodate this request.

If you hold a trademark to this name and believe there is a clear case of confusion between the organization name and your trademark, please review our [Trademark Policy](https://docs.github.com/articles/github-trademark-policy/) to determine if you should submit a trademark policy violation report.

Thank you!
